README.md

```diff
@@ -141,7 +143,8 @@ font              | Arial     | Font-family of the text
     width: 1,
     fill: "#000000",
     color: "#ffffff",
-    font: "Calibri"
+    font: "Calibri",
+    anchor: "middle"
```

Comment:
Minor: comma at end?

Reply:
I've kept it consistent with the other examples in the readme